packages/completer/src/manager.ts

```diff
@@ -46,6 +51,13 @@ export class CompletionProviderManager implements ICompletionProviderManager {
     return this._activeProvidersChanged;
   }
 
+  /**
+   * Signal emitted when a selection is made from a completer menu.
+   */
+  get selected(): ISignal<ICompletionProviderManager, ICompleterSelection> {
```

Comment:
Would it be easy to add a simple unit test for this signal being emitted in:

https://github.com/jupyterlab/jupyterlab/blob/7ae2d436fc410b0cff51042a3350ba71f54f4445/packages/metapackage/test/completer/manager.spec.ts#L105

?

Reply:
ok, so after writing my comment saying no it started bothering me that I gave up half way and decided to just push through with the copying code from the surrounding text files and managed to write a 25 line test. It's pushed feel free to review